test/suite/test_checkpoint04.py

```diff
@@ -34,7 +34,7 @@
 from wtdataset import SimpleDataSet
 
 class test_checkpoint04(wttest.WiredTigerTestCase):
-    conn_config = 'cache_size=50MB,statistics=(all)'
+    conn_config = 'cache_size=50MB,statistics=(all,clear)'
```

Comment:
Can you add a comment saying you are using `clear` to reset stats? You can use the previous comment that used to be there:
```
We don't want stats from earlier runs to interfere with later runs
```

Reply:
done! [2b6b66a](https://github.com/wiredtiger/wiredtiger/pull/9732/commits/2b6b66afcde3d290d674903790895a3298cfab81)